instrumentation/google.golang.org/grpc/otelgrpc/test/grpc_stats_handler_test.go

```diff
@@ -579,3 +588,49 @@ func checkServerSpans(t *testing.T, spans []trace.ReadOnlySpan) {
 		otelgrpc.GRPCStatusCodeKey.Int64(int64(codes.OK)),
 	}, pingPong.Attributes())
 }
+
+func checkServerRecords(t *testing.T, reader metric.Reader) {
+	rm := metricdata.ResourceMetrics{}
+	err := reader.Collect(context.Background(), &rm)
+	assert.NoError(t, err)
+	require.Len(t, rm.ScopeMetrics, 1)
+	require.Len(t, rm.ScopeMetrics[0].Metrics, 5)
+	for _, m := range rm.ScopeMetrics[0].Metrics {
+		require.IsType(t, m.Data, metricdata.Histogram[int64]{})
+		data := m.Data.(metricdata.Histogram[int64])
+		for _, dpt := range data.DataPoints {
```

Comment:
Should buckets be validated to ensure the correct delta of time has been calculated?

Reply:
Oh nevermind, I guess we don't have control of the timestamps or don't have pluggable clocks for testing.